omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -20,7 +20,5 @@
   		    http://www.springframework.org/schema/util
   		    http://www.springframework.org/schema/util/spring-util-3.0.xsd">
 
-	<bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping"/>
-	
-	<context:component-scan base-package="org.openmrs.module.episodes.web.controller" />
+	<context:component-scan base-package="org.openmrs.module.episodes" />

```

Comment:
why not make it narrower? "org.openmrs.module.episodes.web"

Reply:
then the component scan should be in the API module moduleApplicationContext.xml, no? also, I would have preferred to add a ".api" package and pushing all daos, services under that. I am ok right now, if you move this to the API applicationcontext

Reply:
Created the tech debt card [https://bahmni.atlassian.net/browse/BAH-1891](https://bahmni.atlassian.net/browse/BAH-1891)